docs/security.md

```diff
@@ -14,6 +14,8 @@ This document outlines our security policy for the codebase, and how to report v
 
```

Comment:
I could be wrong, but doesn't moving this file break the GitHub UI for security policy?

Reply:
They say, it won't: https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository#about-security-policies